parse-gemspec

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
```

Comment:
I reckon the README should call out that this action needs ruby installed, it's kind of obvious but make it very obvious imo

Reply:
Ok, I'll add to the readme to say the rubysetup stuff is required.